instrumentation/grape/lib/opentelemetry/instrumentation/grape/instrumentation.rb

```diff
@@ -41,11 +34,13 @@ def gem_version
         end
 
         def require_dependencies
-          require_relative 'handler'
+          require_relative 'subscriber'
+          require_relative 'event_handler'
+          require_relative 'custom_subscribers/endpoint_run'
```

Comment:
do we need these here ? shouldn't they be pulled in by requiring `subscriber` ?

Reply:
Indeed, I could move the `require_relative` to the subscriber.
The reason why I chose to include it here was because in all other instrumentations all the `requires` were inside this method, so I decided to do the same here for consistency.

In any case, I'm fine either way, so let me know which one you would prefer 😄

Reply:
>The reason why I chose to include it here was because in all other instrumentations all the requires were inside this method, so I decided to do the same here for consistency.

I would prefer the requires be present where they are required (unintentional 🙈 ).

I think just the require_relative for subscriber should remain

Reply:
I'll change it then 😄

Reply:
I'm interested to see what the maintainers say on this.

I imagine the intent is to stop any extraneous loading of files. So, as long as the require_relatives are in the right place, then in theory there is no problem. If I'm not mistaken, that would only be shifting the `require_relative 'custom_subscribers/endpoint_run'` to `event_handler.rb`, but you're closer to the code to know the real answer 😄

Reply:
I changed it already, let's see what the maintainers say 🍿